app/celery/scheduled_tasks.py

```diff
@@ -533,3 +535,29 @@ def weekly_dwp_report():
         ),
         due_at=convert_utc_to_bst(datetime.utcnow()) + timedelta(days=7),
     )
+
+
+@notify_celery.task(bind=True, name="change-dvla-password", max_retries=55, retry_backoff=True, retry_backoff_max=300)
+def change_dvla_password(self):
+    try:
+        dvla_client.change_password()
+    except LockError:
+        # some other task is currently changing the api key. let that process handle it and quietly exit
+        current_app.logger.info("change-dvla-api-key lock held by other process, doing nothing")
+    except DvlaRetryableException:
+        # could be
```

Comment:
the possibilities are endless

Reply:
(but I'm not sure these comments are super useful)

Reply:
ah but they _could be_ useful

(ill squash em out 🦶 🐛 )